spec/models/paper_trail/version_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe PaperTrail::Version, versioning: true do
+  let(:hospital) { create(:hospital, name: "nome hospital", address: "Barbalha- CE") }
+
+  context "when persist object in database" do
+    it "does not create version" do
+      expect(hospital).to be_persisted
+      expect(hospital.versions.count).to eq(0)
+    end
+  end
+
+  context "when update object in database" do
+    let(:update_hospital) { hospital.update!(name: "Hospital") }
+    let(:hospital_version) { hospital.versions }
+    let(:last_hospital_version) { hospital_version.last }
+
+    before { update_hospital }
+
+    it { expect(hospital_version.count).to eq(1) }
+    it { expect(last_hospital_version.object["name"]).to eq("nome hospital") }
+  end
+
+  context "when api request", type: :request do
```

Comment:
I feel like this context could be at the hospital request specs or in a isolated papertrail request spec

Reply:
It makes sense, I'll add an isolated papertrail request spec.  [Commit hash](https://github.com/espoo-dev/med_system_backend/pull/175/commits/f38fc8b3b3c26b23c87bbae61cb3449711901c98) `f38fc8b3b3c26b23c87bbae61cb3449711901c98`